src/main/java/com/cdancy/jenkins/rest/JenkinsClient.java

```diff
@@ -100,6 +123,10 @@ public Properties overrides() {
         return this.overrides;
     }
 
+    public List<Module> modules() {
+        return this.modules;
+    }
```

Comment:
I'm going to push back against exposing this unless we have a really good reason to do so.